content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -109,6 +109,7 @@ different Kubernetes components.
 | `ExpandPersistentVolumes` | `true` | Beta | 1.11 | |
 | `ExperimentalHostUserNamespaceDefaulting` | `false` | Beta | 1.5 | |
 | `GenericEphemeralVolume` | `false` | Alpha | 1.19 | |
+| `GracefulNodeShutdown` | `false` | Alpha | 1.20 | |
```

Comment:
Please don't forget to add a description down below that explains what functionality this feature provides

Reply:
Done.